### PR TITLE
Cleanup `eduvulcan.pl` home and login page

### PR DESCRIPTION
**Pull request by Marioneq4958**

### CODE REVIEW

**Comment by Marioneq4958** [comment]
- Add login button in home
- Hide eduVulcan features
- Hide eduVulcan app download links
- Hide eduVulcan banner
- Hide maziaj

**Review by PanLiszka — CHANGES_REQUESTED** [review]
opera_BuTfpbuwh8

It removes buttons that gives ability to get into register.

Normally it looks like that:

opera_kCAgaJ4x1e

**Review by PanLiszka — CHANGES_REQUESTED** [review]
You removed ability to get into e-register

File: patches/cleanUpEduVulcanHome.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* eduVulcan features, app downlaod links, eduvulcan banner, maziaj */
+.part-constant:has(.carousel-area-no-authenticated), .box-app, .carousel-component, .ribbon-bg {
+    display: none !important;
```
Comment (PanLiszka) [inline patches/cleanUpEduVulcanHome.css:3]: opera_SbFdD261ES

Here you removes the part that gives me access to e-register

**Review by PanLiszka — CHANGES_REQUESTED** [review]

File: patches/cleanUpEduVulcanHome.css (hunk 1 above)
Comment (Marioneq4958) [reply patches/cleanUpEduVulcanHome.css:3]: It hides only for .part-constant**:has(.carousel-area-no-authenticated)**. Can you show HTML?

File: manifest.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "*://dziennik-uczen.vulcan.net.pl/*",
     "*://dziennik-wiadomosci.vulcan.net.pl/*",
     "*://uczen.eduvulcan.pl/*",
-    "*://wiadomosci.eduvulcan.pl/*"
+    "*://wiadomosci.eduvulcan.pl/*",
+    "*://eduvulcan.pl/*"
```
Comment (banocean) [inline manifest.json:13]: This will result in some patches breaking

**Review by PanLiszka — APPROVED** [review]
Now looks good! :)

**Comment by banocean** [comment]
> Now looks good! :)

May I see how it looks like on desktop?

**Comment by PanLiszka** [comment]
> > Now looks good! :)
> 
> May I see how it looks like on desktop?

opera_7MoKI1wx0y
opera_rHRIujJnBL

File: patches/addEduVulcanLoginButton.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+if (!document.querySelector(".icon-logout"))
+  document.querySelector(".part-business").innerHTML =
+    '<a class="link-button button-primary" style="display: block;margin-top: 20px;" href="/logowanie">Zaloguj się</a>' +
+    document.querySelector(".part-business").innerHTML;
```
Comment (banocean) [inline patches/addEduVulcanLoginButton.js:4]: co ta linijka ma na celu właściwie?

Comment (banocean) [inline patches/addEduVulcanLoginButton.js:4]: using `document.createElement` and `insertBefore` seems like more proper way of doing it

**Comment by banocean** [comment]
> > > Now looks good! :)
> > 
> > 
> > May I see how it looks like on desktop?
> 
> opera_7MoKI1wx0y opera_rHRIujJnBL

Whatever is going on with the alignment of the element under the button, it needs to be fixed too (@Marioneq4958)

**Comment by Marioneq4958** [comment]
> > > > Now looks good! :)
> > > 
> > > 
> > > May I see how it looks like on desktop?
> > 
> > 
> > opera_7MoKI1wx0y opera_rHRIujJnBL
> 
> Whatever is going on with the alignment of the element under the button, it needs to be fixed too (@Marioneq4958)

Fixed :)

obraz

**Review by banocean — CHANGES_REQUESTED** [review]
Seems like other patches loads on a login page, I'd add checks with `doesRunHere` field in new patches API to them to avoid that

**Review by PanLiszka — APPROVED** [review]
opera_h1OUQbn95z

Seems to be good. I love that it automaticaly redirects you to login page

**Pull request merged**